docs/content/en/docs/snippets/tasks/install.md

```diff
@@ -1,59 +0,0 @@
-# Installation Instructions
```

Comment:
was only linked in the readme and that link was changed to point to the klt docs page about installation so this file can be removed now